### PR TITLE
Cypress-IDs and prettier for tests

### PR DESCRIPTION
**Pull request by ErikMichelson**
### Component/Part
E2E Tests

### Description
This PR adds a utility method `cy.getById` to avoid writing the `[data-cypress-id=""]` selector every time. It also adds cypress-ids to components which are tested but didn't have such an id.
Furthermore, this PR enables prettier to also run in the cypress directory.

### Steps

- [x] Added implementation
- [x] Added / updated tests
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### Related Issue(s)
none

### CODE REVIEW

File: cypress/integration/editorMode.spec.ts [hunk 1]
```diff
@@ -7,25 +7,18 @@
 describe('Editor mode from URL parameter is used', () => {
   it('mode view', () => {
     cy.visitTestEditor('view')
-    cy.get('.splitter.left')
-      .should('have.class', 'd-none')
-    cy.get('.splitter.right')
-      .should('not.have.class', 'd-none')
+    cy.get('.splitter.left').should('have.class', 'd-none')
+    cy.get('.splitter.right').should('not.have.class', 'd-none')
```
Comment (mrdrogdrog) [inline cypress/integration/editorMode.spec.ts:11]: maybe replace this with "should not be visible"?

File: cypress/integration/shortcodes.spec.ts [hunk 2]
```diff
@@ -21,26 +21,21 @@ describe('Short code gets replaced or rendered: ', () => {
...
+      cy.getMarkdownBody().find('a').should('have.attr', 'href', 'https://speakerdeck.com/example/123456789')
     })
   })
 
   describe('youtube', () => {
     it('renders click-shield', () => {
       cy.setCodemirrorContent(`{%youtube YE7VzlLtp-4 %}`)
-      cy.getMarkdownBody()
-        .find('[data-cypress-id="click-shield-youtube"]')
+      cy.getMarkdownBody().find('[data-cypress-id="click-shield-youtube"]')
```
Comment (mrdrogdrog) [inline cypress/integration/shortcodes.spec.ts:38]: use getById

File: src/components/profile-page/access-tokens/profile-access-tokens.tsx [hunk 3]
```diff
@@ -120,7 +122,12 @@ export const ProfileAccessTokens: React.FC = () => {
                       />
                     </Col>
                     <Col xs='auto'>
-                      <IconButton icon='trash-o' variant='danger' onClick={() => selectForDeletion(token.created)} />
+                      <IconButton
+                        icon='trash-o'
+                        variant='danger'
+                        onClick={() => selectForDeletion(token.created)}
```
Comment (mrdrogdrog) [inline src/components/profile-page/access-tokens/profile-access-tokens.tsx:128]: useCallback

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: cypress/integration/editorMode.spec.ts (hunk 1 above)
Comment (ErikMichelson) [reply cypress/integration/editorMode.spec.ts]: done

File: cypress/integration/shortcodes.spec.ts (hunk 2 above)
Comment (ErikMichelson) [reply cypress/integration/shortcodes.spec.ts]: done

File: src/components/profile-page/access-tokens/profile-access-tokens.tsx (hunk 3 above)
Comment (ErikMichelson) [reply src/components/profile-page/access-tokens/profile-access-tokens.tsx:128]: The complete profile page uses this messy old code that ignores the fact that useCallback exists.
Guess why there's a branch `refactor-profile-page` basing on top of this PR. :D

**Review by mrdrogdrog — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]

**Pull request merged**